code/modules/atmospherics/machinery/components/binary_devices/dp_vent_pump.dm

```diff
@@ -154,6 +154,10 @@
 	if(!signal.data["tag"] || (signal.data["tag"] != id) || (signal.data["sigtype"]!="command"))
 		return
 
+	if(("status" in signal.data)) //Send stauts and early return, I'm cargoculting the timer here.
+		addtimer(CALLBACK(src, PROC_REF(broadcast_status)), 2)
```

Comment:
y doth this need a timer, on something like signal receive this could cause alot of timer churn

Reply:
> I'm cargoculting the timer here.

I can remove it for next test if you'd like

Reply:
pls

Reply:
```suggestion
	if(("status" in signal.data)) //Send stauts and early return, I'm cargoculting the timer here.
		broadcast_status()
```